src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -545,5 +652,606 @@ private void DestroyHandles()
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ProviderSetName(string name)
+        {
+            int error = Interop.SensorProvider.ProviderSetName(_sensorProviderHandle, name);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in setting name of sensor provider");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.ProviderSetName Failed");
+            }
+
+        }
+
+        /// <summary>
+        /// Sets the vendor to the sensor provider.
+        /// </summary>
+        /// <param name="vendor">
+        /// The vendor of the sensor.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ProviderSetVendor(string vendor)
+        {
+            int error = Interop.SensorProvider.ProviderSetVendor(_sensorProviderHandle, vendor);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in setting vendor of sensor provider");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.ProviderSetVendor Failed");
+            }
+
+        }
+
+        /// <summary>
+        /// Sets the range of possible sensor values to the sensor provider.
+        /// </summary>
+        /// <param name="minRange">
+        /// The lower bound.
+        /// </param>
+        /// <param name="maxRange">
+        /// The upper bound.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ProviderSetRange(float minRange, float maxRange)
+        {
+            int error = Interop.SensorProvider.ProviderSetRange(_sensorProviderHandle, minRange, maxRange);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in setting range of sensor provider");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.ProviderSetRange Failed");
+            }
+
+        }
+
+        /// <summary>
+        /// Sets the resolution of sensor values to the sensor provider.
+        /// </summary>
+        /// <param name="resolution">
+        /// The resolution.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ProviderSetResolution(float resolution)
+        {
+            int error = Interop.SensorProvider.ProviderSetResolution(_sensorProviderHandle, resolution);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in setting resolution of sensor provider");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.ProviderSetResolution Failed");
+            }
+
+        }
+
+        /// <summary>
+        /// Registers the callback function to be invoked when a listener starts the sensor provider.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void ProviderSetStartCb()
```

Comment:
What is it?
Do not USE callback

Reply:
This API registers the callback function to be invoked when a listener starts the sensor provider.